.devcontainer/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM centos/devtoolset-6-toolchain-centos7:latest
```

Comment:
devtoolset-6 is deprecated now as well - I don't think it's even in the repos anymore.  Are you able to move to a newer one or is that not feasible?

Reply:
@bnhhughes we want to change as little as possible for now. we will make some updates and then migrate to a newer gcc.

Reply:
@bnhhughes Do you know when they plan to pull the plug entirely on devtoolset-6?

Reply:
I think they already have - we were using it in a chroot for a while and last time we went to build one from scratch we couldn't even pull down devtoolset-6 packages.  If you go to the Software Collections page for devtoolset-6 it points you to the repo, but the repo starts with devtoolset-7 (http://mirror.centos.org/centos/7/sclo/x86_64/rh/Packages/d/).  We've moved to devtoolset-9 which gives you GCC 9.3.1